uniffi_bindgen/src/bindings/python/templates/wrapper.py

```diff
@@ -23,6 +23,8 @@
 
 {% include "RustBufferHelper.py" %}
 
+{% include "RustErrorTemplate.py" %}
```

Comment:
@tarikeshaq @rfk I think Python doesn't allow the usage of a generic class-type `ctypes.POINTER(ctypes.Structure)` on args list. I can't make something like `argtypes = (ctypes.POINTER(ctypes.Structure),)` (and send a class which extends `ctypes.Structure`)

I made this file to avoid this. A `RustError` class is declared on top of file to get the Rust result across FFI.
You guys know a better way to solve this? 👀

Reply:
> Not too familiar with what Python allows here, but this feels okay to me, and we can iterate on it if need be 😄 Also feel free to have this as a part of `ErrorTemplate.py` if you don't wanna add a new file.

It's a good idea!